app/views/layouts/_head.html.erb

```diff
@@ -41,7 +41,7 @@
 
   // Interaction tracking
   $(document).ready(function(e) {
-    TrackLinks( $( '.region a' ) );
+    TrackLinks( $( 'a' ) );
```

Comment:
The expansion will end up tracking all interactions with links, yes - so in the case where a link also results in a page load within Bento, that page load will be counted as part of the page load approach as well as recorded as an Event. IMO, this is only problematic if we take the step of adding event counts and page load counts, which I would argue is bad practice even without this step.